gems/pending/openstack/events/openstack_rabbit_event_monitor.rb

```diff
@@ -42,7 +42,7 @@ def self.test_connection(options = {})
       $log.info("#{log_prefix} The Openstack AMQP service may be using a different provider."\
                 " Enable debug logging to see connection exception.") if $log
       $log.debug("#{log_prefix} Exception: #{e}") if $log
-      return false
+      raise e
```

Comment:
Yeah, I was thinking that that would make sense, though the excon stuff was already exposed like this,